central/deployment/cache/cache.go

```diff
@@ -11,10 +11,30 @@ const (
 )
 
 var (
-	cache = expiringcache.NewExpiringCache(deletedDeploymentsRetentionPeriod)
+	cache = &deploymentCache{
+		cache: expiringcache.NewExpiringCache(deletedDeploymentsRetentionPeriod),
+	}
 )
 
-// DeletedDeploymentCacheSingleton returns a global expiringcache for deployments that have been recently deleted
-func DeletedDeploymentCacheSingleton() expiringcache.Cache {
+type DeletedDeployments interface {
+	Add(id string)
+	Contains(id string) bool
+}
+
+type deploymentCache struct {
+	cache expiringcache.Cache
+}
+
+func (c *deploymentCache) Add(id string) {
+	cache.cache.Add(id, struct{}{})
+}
+
+func (c *deploymentCache) Contains(id string) bool {
+	_, ok := cache.cache.Get(id)
+	return ok
+}
+
+// DeletedDeploymentsSingleton returns a global expiringcache for deployments that have been recently deleted
+func DeletedDeploymentsSingleton() DeletedDeployments {
```

Comment:
Any reason behind this rename (removal of the Cache word) ?

Reply:
It's in cache package so usage will be cache.DeletedDeoloynent so I think it's better with just a single cache